content/security.md

```diff
@@ -249,17 +271,23 @@ You can tune more finely this permission by restricting or allowing the access t
 | List families | GET on `/families` and on `/families/{family_code}`|
 | List attributes | GET on `/attributes` and on `/attributes/{attribute_code}` |
 | List attribute options | GET on `/attributes/{attribute_code}/options` and on `/attributes/{attribute_code}/options/{attribute_option_code}` |
-| List channels | GET on `/channels` |
-| List locales | GET on `/locales` |
+| List attribute group *(2.0 only)* | GET on `/attribute-groups` and on `/attributes-groups/{attribute_groups_code}` |
+| List association types *(2.0 only)* | GET on `/association-types` and on `/association-types/{association_type_code}` |
+| List channels | GET on `/channels` and on `/channels/{channel_code}` |
+| List locales | GET on `/locales` and on `/locales/{locale_code}` |
+| List currencies *(2.0 only)*  | GET on `/currencies` and on `/currencies/{currency_code}` |
 | Create and update categories | POST and PATCH on `/categories/{category_code}` <br/> PATCH on `/categories` |
 | Create and update families | POST and PATCH on `/families/{family_code}` <br/> PATCH on `/families` |
 | Create and update attributes | POST and PATCH on `/attributes/{attribute_code}` <br/> PATCH on `/attributes`|
-| Create and update attribute options | POST and PATCH on `/attributes/{attribute_code}/options/{attribute_option_code}` |
+| Create and update attribute options | POST and PATCH on `/attributes/{attribute_code}/options/{attribute_option_code}` <br/> PATCH on `/attributes/{attribute_code}/options` |
+| Create and update attribute groups *(2.0 only)* | POST and PATCH on `/attribute-groups/{attribute_group_code}` <br/> PATCH on `/attribute-groups` |
+| Create and update association types *(2.0 only)* | POST and PATCH on `/association-types/{association_type_code}` <br/> PATCH on `/association-types` |
+| Create and update channels *(2.0 only)* | POST and PATCH on `/channels/{channel_code}` <br/> PATCH on `/channels` |
 
 :::warning
 As accessing the API grants higher privileges than when using the UI, we strongly recommend creating one or more dedicated users with specific roles for the web API.
 :::
 
 :::info
-Note that if a role has `Overall Web API` access, then it means that the users depending on that role will be able to make requests on products. There is no way to only restrict the access on products for now. 
+Note that if a role has `Overall Web API` access, then it means that the users depending on that role will be able to make requests on products. There is no way to only restrict the access on products, except if you are using a 2.0 Entreprise Edition, the EE permissions based on user groups being applied on the API.
```

Comment:
`it means that all the users `


`There is no way to only restrict the access on products, except if you are using a 2.0 Entreprise Edition. In this case, the EE permissions based on user groups being applied on the API.`